src/main/java/uk/gov/hmcts/reform/prl/services/cafcass/CafcassUploadDocService.java

```diff
@@ -58,8 +58,6 @@ public void uploadDocument(String authorisation, MultipartFile document, String
             }
             CaseData caseData = CaseUtils.getCaseData(caseDetails, objectMapper);
 
-            checkIfNumberOfAttachmentExceeds(caseData);
```

Comment:
is this method used anywhere else in the code. if not, then can we remove the method as well?

Reply:
Also, have you tested the change whether it is allowing to upload more than 3 files for a case

Reply:
> Also, have you tested the change whether it is allowing to upload more than 3 files for a case

Tested, works fine